packages/perseus/src/util/extract-perseus-data.ts

```diff
@@ -599,7 +611,8 @@ export const getCorrectAnswerForWidgetKey = (
     itemData: PerseusItem,
 ): string | undefined => {
     const rubric = itemData.question.widgets[widgetKey].options;
-    const widgetType = getWidgetTypeFromWidgetKey(widgetKey);
+    const widgetMap = getWidgetsMapFromItemData(itemData) as PerseusWidgetsMap;
+    const widgetType = getWidgetTypeByWidgetId(widgetKey, widgetMap) as string;
```

Comment:
Question, nb: Is the type declaration here because the function could also be null? The function on line 614 already returns something of type PerseusWidgetsMap though, right?

Reply:
Line 614: getWidgetsMapFromItemData used to return `PerseusWidgetsMap | null` and now returns `PerseusWidgetsMap`. I forgot to remove the `as`.

Line 615: while unlikely, it isn't beyond the realm of possibility that we have `[[radio 1]]` and no `"radio 1"` key in `widgets`. I thought it would be best to make sure consumers handle that possibility. However there's already code that exists that _don't_ handle that possibility. Rather than refactor all their logic (which would be scope creep), I just cast for now.